ci/azure-pipelines-sqlalchemy-compatibility.yml

```diff
@@ -98,25 +98,24 @@ stages:
              python -c "import great_expectations as gx; print('Successfully imported GX Version:', gx.__version__)"
            displayName: 'Import Great Expectations'
 
-  - stage: sqlalchemy_compatibility
+  - stage: sqlalchemy_and_pandas_compatibility
     dependsOn: [scope_check, lint, import_ge]
     pool:
       vmImage: 'ubuntu-latest'
 
     jobs:
-      - job: sqlalchemy_compatibility_postgres
+      - job: compatibility_postgres
```

Comment:
I think we should discuss whether we do this compatibility testing with every dialect (I'm not sure). But I think we should at least add one of the dialects that requires a persisted connection for temp tables (like sqlite). What do you think?

Reply:
I think you're absolutely right. Let's talk more about this.